.github/workflows/lint.yml

```
@@ -37,11 +37,10 @@ jobs:
 
       - name: "Setup: all workspaces"
         working-directory: frontend
-        run: npm run codegen --workspaces --if-present
+        run: |
+          npm run codegen --workspaces --if-present
+          npm run intl-compile --workspaces --if-present
 
       - name: "Run ESlint: all workspaces"
         working-directory: frontend
-        # Temporarily ignore until we fix existing issues.
-        # See: https://github.com/GCTC-NTGC/gc-digital-talent/issues/2043
-        continue-on-error: true
         run: npm run lint --workspaces --if-present
```

If we want `npm run lint` to fail if there are even any warnings, we can use this, which passed the flag through `npm run` down to `eslint`.

```suggestion
        run: npm run lint --workspaces --if-present -- --max-warnings=0
```

---

Added to todos to spin out another issue. no action here